### DISCUSSION TITLE
Cannot print on paper a PDF generated by Quarto using adobe reader on windows PC

### DISCUSSION THREAD

**Discussion by QjuanQ** [quarto-dev/quarto-cli#9797] [Q&A]
### Description

After working for quite a long time with my data, I get a beautiful pdf report using RSTUDIO and quarto, everything is smooth and fine, ultil I have to print the report. Using de RSTUDIO default viewer there is no problem printing, but with adobe pdf reader it opens the document and I can see all of it, but when printing,  it gets stuck and the progress bar doesn't progress at all...it says flattening... And nothing comes out of the printer (that otherwise is working propperly). There are ggplot2 images and some tables inserted, that are perfectly visualized in the pdf, but no way to print it from adobre reader, if I remove the tables and plots all goes fine, but then de report is useless :)

I don't think it is a R code problem, as the result of the pdf report is the right one visually, but a Quarto or tynitex Code problem, that IS generating a PDF doc not compatible with adobe itself.

I tried to change the latex from tinytex to miktex, I have change the latex engine manually, I haver remove all the options set in header so that all defaults were use, I've tried Quarto, Rmarkdown, and the same result again and again. I simplified the plot, I've chage the table from kable to kable extra to pander tables, and nothing changed.

Quarto 1.6
R 4.4.0
Tinytext 0.51

I generate more than 900 reports, so handmade solutions are out of scope.

¿Any suggestions?

Thanks

**Comment by cderv** [comment]
I am sorry to hear your experience.

Regarding producing PDF, what Quarto is doing with the help of Pandoc is to produce a `.tex` file. Then this TeX file is passed to LaTeX for rendering. Quarto only calls LaTeX for you. 

Did you look at the `.tex` file and tried to find what caused issue ? If anything is invalid after convertion to PDF, then this is more related to how LaTeX build PDF. 

I don't know about adobe specificity, but if the PDF rendered looks good, then it could be a printing problem from Adobe side. From Quarto side, we don't control the PDF rendering, nor what you include in your document. I know there could be a bazillion things going on while doing PDF, from the size of the table, the plots themselves, the fonts used, the graphics formats, ... Some of this configuration can be set by Quarto, and maybe some default of Quarto does not play well with your content. But most of this is dependant on the content. 

You did not provide any  content to look at, so this is as far as I can answer. I can just guess and give hints; If you can share a reproducible example, or even the PDF produced, we could try help (or someone from the community). 

> it says flattening...

I don't know much about adobe by Quick google search give us some clues

Flattening in Adobe is about transparent background I believe. 
* https://helpx.adobe.com/acrobat/kb/printing-complex-pdfs-acrobat.html#main_Adjust_the_flattening_of_transparent_objects_

It seems to be common issue in Adobe for some PDF having specific contents. 
* https://community.adobe.com/t5/acrobat-reader-discussions/pdf-flattening-dialogue-when-printing-pdfs-that-are-coverted-from-powerpoints/td-p/4065622
* https://www.cadzation.com/help/PrintAsImage.htm

So I would really check the content you have in your document, the plots produced and the format used. We don't know if you are default pdf format options, or if you tweak, or how you generate the figures, or which type you include. 

Again, provide some example we can work with, this would be really valuable help for us to help you further. 

Hope this helps. 

Thanks !

**Reply by QjuanQ** [reply]
I understand and appreciate very much your answer.I Will provide some example, but I have to arrange It with fake data, as I'm using real data so far.
Is there a way to force laxex to use a previous versión of PDF, let's say 1.3?

**Reply by cderv** [reply]
> Is there a way to force laxex to use a previous versión of PDF, let's say 1.3?

I don't know this. Sorry
